common/config.py

```diff
@@ -0,0 +1,33 @@
+from configparser import ConfigParser
+from typing import TypeVar
+from django.utils.encoding import force_text
+
+
+class CredentialsLoader:
```

Comment:
Generally look at common/config.py here: https://github.com/Xcov19/covidX/pull/91/files
This is a better way to do what I've suggested below with interfaces;